community/kernel/src/test/java/org/neo4j/test/LogTestUtils.java

```diff
@@ -128,6 +131,63 @@ public static void assertLogContains( FileSystemAbstraction fileSystem, String l
         }
     }
 
+    /**
+     * Opens a {@link LogEntryCursor} over all log files found in the storeDirectory
+     *
+     * @param fs {@link FileSystemAbstraction} to find {@code storeDirectory} in.
+     * @param storeDirectory the store directory where the log files are.
+     * @param logVersionCallback will be kept up to date with which log version the cursor is at.
+     */
+    public static LogEntryCursor openLogs( final FileSystemAbstraction fs, File storeDirectory,
+            final AtomicLong logVersionCallback )
+    {
+        StoreChannel channel = null;
```

Comment:
Can `channel` be moved into try-with-resources?
I think this should be possible if `logFiles` variable move up.